azure-pipelines/build.yml

```diff
@@ -1,4 +1,11 @@
 steps:
+
+- task: UseDotNet@2
+  displayName: 'Use .NET Core 2 sdk'
```

Comment:
Either add a comment or update the title to indicate why this is needed

Reply:
Added, this is used by ESRP code signing.